tests/qunit/js/util.js

```diff
@@ -375,6 +375,7 @@ define(['exports', 'jquery', 'underscore', 'oae.core', 'jquery.properties-parser
                 '/shared/oae/macros/list.html': null,
                 '/ui/content.html': null,
                 '/ui/discussion.html': null,
+                '/ui/emailverification.html': null,
                 '/ui/folder.html': null,
                 '/ui/group.html': null,
                 '/ui/index.html': null,
```

Comment:
- `api.admin.js` is missing from `apiJS`
- `api.push.js` is missing from `apiJS`